src/goto-symex/goto_symex_state.cpp

```diff
@@ -493,9 +499,9 @@ void goto_symex_statet::rename(
   }
   else if(expr.id()==ID_address_of)
   {
-    assert(expr.operands().size()==1);
+    PRECONDITION(expr.operands().size()==1);
     rename_address(expr.op0(), ns, level);
```

Comment:
I'd say DATA_INVARIANT because this is something that should always be true of a well formed address-of